src/plugins/application_config/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  root: true,
+  extends: ['@elastic/eslint-config-kibana', 'plugin:@elastic/eui/recommended'],
+  rules: {
+    '@osd/eslint/require-license-header': 'off',
```

Comment:
Why do we want to turn off require license header?

Reply:
The file was automatically added by the plugin generator helper. Just check that other plugins do not have this file. Let me just remove it. Thanks for the scrutiny!